beaver/transports/stomp_transport.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+import stomp
+
+from beaver.transports.base_transport import BaseTransport
+from beaver.transports.exception import TransportException
+
+
+class StompTransport(BaseTransport):
+
+    def __init__(self, beaver_config, logger=None):
+        """
+        Mosquitto client initilization. Once this this transport is initialized
+        it has invoked a connection to the server
+        """
+        super(StompTransport, self).__init__(beaver_config, logger=logger)
+
+        self.createConnection(beaver_config)
+        self.logger = logger
+
+ 
+    def callback(self, filename, lines, **kwargs):
+        """publishes lines one by one to the given topic"""
+        timestamp = self.get_timestamp(**kwargs)
+        if kwargs.get('timestamp', False):
+            del kwargs['timestamp']
+        
+
+        for line in lines:
+            try:
+                import warnings
+                with warnings.catch_warnings():
+                    warnings.simplefilter('error')
+                    m = self.format(filename, line, timestamp, **kwargs)
+                    self.logger.debug("Sending message " + m)
+                    self.conn.send(destination=self.queue, body=m)    
```

Comment:
Hi Jose,

I copied this from the other transports, and they don't seem to do it.

Stomp (like most message interfaces) only sends a message at a time I could introduce transactions around the group of lines so that there is less chatter on the network (i.e. only 1 ack) but this becomes redundant if the messages aren't persistent on the broker.

We could combine the lines into 1 message and get logstash to split them but it could have to be a config option.

also Beaver has the multi-line config which would combine them at source.

I'll be happy to implement it for the stomp transport with a bit of advice from you